search/indexing_api.py

```diff
@@ -190,6 +190,19 @@ def serialize_program_enrolled_user(program_enrollment):
     return serialized
 
 
+def filter_current_work(profile):
+    """
+    Remove work_history objects that are not current
+    """
+    current_employment = []
+    for work in profile['work_history']:
+        if not work['end_date']:
+            current_employment.append(work)
```

Comment:
It looks like this case is uncovered FYI. Can you add a test that asserts that only current work history entries end up in the Elasticsearch index?